docker-image/action.yaml

```diff
@@ -36,3 +39,5 @@ runs:
     - ${{ inputs.score-threshold }}
     - --log-level
     - ${{ inputs.log-level }}
+  env:
+    MONDOO_CONFIG_BASE64: ${{ inputs.service-account-credentials }}
```

Comment:
So when `service-account-credentials` isn't set, then the environment from the caller of the action is used? I guess that's why this works?

Reply:
Yes, that's how I also think it works. I couldn't find explicit docs about it. But having a look at how the two different steps are called, the assumption seems valid:

The first step w/o env var, but w/ arg:
![image](https://user-images.githubusercontent.com/827818/208097349-f30d5eb8-db11-4d8a-9017-cc37fe1f2a4a.png)

The second step w/ env, but w/o arg:
![image](https://user-images.githubusercontent.com/827818/208097462-8e09febc-1f4e-4526-a7cc-c89c8f1977e5.png)

And both seem to report to the same space.